contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,647 @@
+// -----------------------------------
```

Comment:
You can insert `// amarna: disable=arithmetic-add,arithmetic-div,arithmetic-mul,arithmetic-sub` as the first line in here to disable those false Amarna warnings.